### PR TITLE
Allows us to use the model on a given video

### PR DESCRIPTION
**Pull request by reteppeter**
use.py can be called on a given file and pass it through the model. This required some changes to extract_data.py.

### CODE REVIEW

**Comment by reteppeter** [comment]
The full model is now able to be used, though it seems to not be very accurate.

File: use.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
+import os
```
Comment (mt-empty) [inline use.py:1]: Could you put a comment about how to use this file and an example on what arguments it expects
`python use.py arg1 arg2 ...`

**Review by mt-empty — CHANGES_REQUESTED** [review]
All good, other that that one small issue

File: use.py (hunk 1 above)
Comment (reteppeter) [reply use.py:1]: If you pass --help as an argument, the other valid arguments with descriptions are listed. For our current uses no arguments should be given.

**Review by mt-empty — APPROVED** [review]

**Pull request merged**